Jenkinsfile

```diff
@@ -10,84 +10,62 @@
-            sshPut remote: buildServer, from: "source.tar", into: buildDir
-            sshCommand remote: buildServer, command: \
-                """
-                cd ${buildDir} &&
-                pax -r -x tar -o to=IBM-1047 -f source.tar
-                """
+    pipeline.setup(
+        packageName: 'org.zowe.zss',
+        github: [
+            email                      : lib.Constants.DEFAULT_GITHUB_ROBOT_EMAIL,
+            usernamePasswordCredential : lib.Constants.DEFAULT_GITHUB_ROBOT_CREDENTIAL,
+        ],
+        artifactory: [
+            url                        : lib.Constants.DEFAULT_ARTIFACTORY_URL,
+            usernamePasswordCredential : lib.Constants.DEFAULT_ARTIFACTORY_ROBOT_CREDENTIAL,
+        ],
+        pax: [
+            sshHost                    : lib.Constants.DEFAULT_PAX_PACKAGING_SSH_HOST,
+            sshPort                    : lib.Constants.DEFAULT_PAX_PACKAGING_SSH_PORT,
+            sshCredential              : lib.Constants.DEFAULT_PAX_PACKAGING_SSH_CREDENTIAL,
+            remoteWorkspace            : lib.Constants.DEFAULT_PAX_PACKAGING_REMOTE_WORKSPACE,
+        ],
+        extraInit: {
+            def version = [
+                sh(script: "cat build/version.properties | grep PRODUCT_MAJOR_VERSION | awk -F= '{print \$2};'", returnStdout: true).trim(),
+                sh(script: "cat build/version.properties | grep PRODUCT_MINOR_VERSION | awk -F= '{print \$2};'", returnStdout: true).trim(),
+                sh(script: "cat build/version.properties | grep PRODUCT_REVISION | awk -F= '{print \$2};'", returnStdout: true).trim(),
+            ].join('.')
+            pipeline.setVersion(version)
+            echo "Current version is v${version}"
         }
     )
 
     pipeline.build(
         operation: {
-            sshCommand remote: buildServer, command: \
-                """
-                export JAVA_HOME=${JAVA_HOME} &&
-                export ANT_HOME=${ANT_HOME} &&
-                export STEPLIB=${XLC_DATASET} &&
-                cd ${buildDir}/build &&
-                ${ANT_HOME}/bin/ant
-                """
+            echo "Build will happen in pre-packaging"
         }
     )
 
-    pipeline.packaging(
-        name: 'zss',
-        operation: {
-            sshCommand remote: buildServer, command: \
-                """
-                cd ${buildDir} &&
-                mkdir LOADLIB SAMPLIB &&
-                cp -X "//DEV.LOADLIB(ZWESIS01)" LOADLIB/ZWESIS01  &&
-                cp -X "//DEV.LOADLIB(ZWESAUX)" LOADLIB/ZWESAUX  &&
-                cp samplib/zis/* SAMPLIB/  &&
-                cp bin/zssServer ./  &&
-                extattr +p zssServer &&
-                pax -x os390 -w -f zss.pax SAMPLIB LOADLIB zssServer
-                """
-            sshGet remote: buildServer, from: "${buildDir}/zss.pax", into: "zss.pax"
-            sshCommand remote: buildServer, command:  "rm -rf ${buildDir}"
-        }
+    pipeline.test(
```

Comment:
We won't have any tests in the repo.
Do we still the stage? 
We could run some testing in a pre-built docker container, or pulling QA stuff from another repo,
but not sure if it's worth it: too many ingredients. I'd have a separate QA pipeline

Reply:
Currently the test stage is mandatory if we have `publish` stage, so I put a dummy stage here. Otherwise we need to find an option to skip that validation.

Reply:
Will add an option with https://github.com/zowe/jenkins-library/issues/42